### ISSUE TITLE
dreambooth Lora training failure

### ISSUE THREAD

**Issue by yayueLaurazhou** [NVlabs/Sana#250]
I am using the image dataset used in dreambooth paper here https://github.com/google/dreambooth/tree/main/dataset/can

on model **Efficient-Large-Model/Sana_1600M_1024px_BF16_diffusers**

accelerate launch /mnt/sdc/zhouyayue/projects/Sana/train_scripts/train_dreambooth_lora_sana.py \
  --pretrained_model_name_or_path="Efficient-Large-Model/Sana_1600M_1024px_BF16_diffusers" \
  --instance_data_dir="/mnt/sdc/zhouyayue/projects/Sana/dreambooth_data" \
  --output_dir="/mnt/sdc/zhouyayue/projects/Sana/output_dreambooth_lora" \
  --mixed_precision="bf16" \
  --instance_prompt="a photo of a <edfe> can" \
  --resolution=1024 \
  --train_batch_size=1 \
  --gradient_accumulation_steps=4 \
  --use_8bit_adam \
  --learning_rate=1e-6 \
  --report_to="wandb" \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --max_train_steps=800 \
  --validation_prompt="ground level view, low camera angle, a realistic photo of a <edfe> can on terrazo tile floor" \
  --validation_epochs=50 \
  --seed="0" \
  --push_to_hub 

I changed the learning rate from 1e-4 to 1e-6, the result is below 

![Image](https://github.com/user-attachments/assets/fa91a345-a5b0-4b48-91cd-c6d7f5e57a81)

at 100 epochs, this are the validation images 

![Image](https://github.com/user-attachments/assets/fc7f991b-6f4f-4df3-af23-9cf0f01b07ed)

![Image](https://github.com/user-attachments/assets/db5615a8-02ba-4977-b481-636ba2f039ca)

![Image](https://github.com/user-attachments/assets/1998d88e-9421-4e67-a764-d05ec125d8e9)

after 700 epochs, it gives me photos that doesn't even look like a can, not to mention subject specific features, 

![Image](https://github.com/user-attachments/assets/d87b5552-9dec-4d70-8785-4c9a1985e637)

![Image](https://github.com/user-attachments/assets/121314ea-db13-4d10-b48b-9ecf63c5d564)

what is wrong with my training?

**Comment by bghira** [comment]
it's the model, not your training.

**Comment by nanaj96** [comment]
can you told to me can we test our diffuser sana lora dreambooth checkpoints model? do you have any workflow?
